adapters/eea-direct.js

```diff
@@ -2,146 +2,104 @@ import { acceptableParameters, convertUnits } from '../lib/utils';
 import { REQUEST_TIMEOUT } from '../lib/constants';
```

Comment:
Ok, so I tested this out and had mixed results. The first time I was running it on Slovakia which is a smallish source (~200 matching measurements), it ran fine. I compared to data coming from the current process and it matched, so that was all great. However, coming back to Slovakia a bit later, after new data was being provided, I ran into the errors below. Note that I also saw these errors when trying to do the larger sources like `EEA Germany` and `EEA France`. Was able to run and test just the specific source with `node index.js -d -s 'EEA Slovakia'`. This looks like perhaps the stream is losing data during it's processing and it doesn't like that?

```
(node:5994) UnhandledPromiseRejectionWarning: Error: Too few fields: expected 24 fields but parsed 3
    at Object.chunk (/Users/jflasher/workspace/openaq-fetch/node_modules/scramjet/lib/string-stream.js:50:60)
    at ReadableStreamStreamer.ChunkStreamer.parseChunk (/Users/jflasher/workspace/openaq-fetch/node_modules/papaparse/papaparse.js:488:18)
    at ReadableStreamStreamer.<anonymous> (/Users/jflasher/workspace/openaq-fetch/node_modules/papaparse/papaparse.js:789:11)
    at ReadableStreamStreamer._streamData (/Users/jflasher/workspace/openaq-fetch/node_modules/papaparse/papaparse.js:1585:25)
    at ReadableStreamStreamer.<anonymous> (/Users/jflasher/workspace/openaq-fetch/node_modules/papaparse/papaparse.js:808:9)
    at StringStream.<anonymous> (/Users/jflasher/workspace/openaq-fetch/node_modules/papaparse/papaparse.js:1585:25)
    at StringStream.emit (events.js:165:20)
    at endReadableNT (_stream_readable.js:1101:12)
    at process._tickCallback (internal/process/next_tick.js:152:19)
(node:5994) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 16760)
(node:5994) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
(node:5994) UnhandledPromiseRejectionWarning: Error: Too few fields: expected 24 fields but parsed 5
    at Object.chunk (/Users/jflasher/workspace/openaq-fetch/node_modules/scramjet/lib/string-stream.js:50:60)
    at ReadableStreamStreamer.ChunkStreamer.parseChunk (/Users/jflasher/workspace/openaq-fetch/node_modules/papaparse/papaparse.js:488:18)
    at ReadableStreamStreamer.<anonymous> (/Users/jflasher/workspace/openaq-fetch/node_modules/papaparse/papaparse.js:789:11)
    at ReadableStreamStreamer._streamData (/Users/jflasher/workspace/openaq-fetch/node_modules/papaparse/papaparse.js:1585:25)
    at ReadableStreamStreamer.<anonymous> (/Users/jflasher/workspace/openaq-fetch/node_modules/papaparse/papaparse.js:808:9)
    at StringStream.<anonymous> (/Users/jflasher/workspace/openaq-fetch/node_modules/papaparse/papaparse.js:1585:25)
    at StringStream.emit (events.js:165:20)
    at endReadableNT (_stream_readable.js:1101:12)
    at process._tickCallback (internal/process/next_tick.js:152:19)
(node:5994) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 17643)
```

Reply:
It seems that some lines are not full inn that data. In the previous implementation such an entry would be simply omitted - I'll fix it

Reply:
Ha yeah, not surprising. Source data across various adapters often has issues.